node/typings.ts

```diff
@@ -1,3 +1,4 @@
+import { Category } from '@vtex/api/lib/clients/apps/catalogGraphQL/category'
```

Comment:
This seems dangerous, someone can easily change this file name or location on `node-vtex-api` since this is not the module exported API and break this app without knowing. I would recommend importing this client from the root index of `node-vtex-api`

Reply:
That is the current way, we would need to change `node-vtex-api` for this to work.